NewmanPostman/npm-debug.log

```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```

Comment:
Do we need to remove this file from source as well?

Reply:
Done, please consider PR#49